modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgniteDataStorageMetricsSelfTest.java

```diff
@@ -389,6 +390,81 @@ public void testWalWrittenBytes() throws Exception {
         assertEquals(exp, ((LongAdderMetric)dsMetricRegistry(n).findMetric("WalWrittenBytes")).value());
     }
 
+    /**
+     * Check whether Wal is reporting correct usage.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testWalTotalSize() throws Exception {
```

Comment:
As I understand it is expected here without getting the segments into the archive? then you need to write about it and make a check.